### PR TITLE
Add macrumors

### PR DESCRIPTION
**Pull request by mbtrs**

### CODE REVIEW

**Comment by mbtrs** [comment]
Close #1176

File: _data/sites.json [hunk 1]
```diff
@@ -6948,6 +6948,16 @@
             "account.lyft.com"
         ]
     },
+    
+    {
```
Comment (tupaschoal) [inline _data/sites.json:6952]:
```suggestion

    {
```

File: _data/sites.json [hunk 2]
```diff
@@ -6948,6 +6948,16 @@
...
+        "name": "Macrumors",
+        "url": "https://macrumors.zendesk.com/hc/en-us/articles/201260797-How-do-I-cancel-my-account-",
+        "notes": "There is no account deletion process offered. Remove your account's Personal Details (Location, About You, etc.), your forum signature, and any social media names in the Contact Details section and leave the account dormant.",
+        "difficulty": "impossible",
```
Comment (tupaschoal) [inline _data/sites.json:6956]:
```suggestion
        "difficulty": "impossible", 
        "notes": "There is no account deletion process offered. Remove your account's Personal Details (Location, About You, etc.), your forum signature, and any social media names in the Contact Details section and leave the account dormant.",
```

This is just being picky, but notes tend to be bigger, so I like the difficulty to appear first :P

**Review by tupaschoal — COMMENTED** [review]
Just some minor details

File: _data/sites.json [hunk 3]
```diff
@@ -6949,6 +6949,16 @@
         ]
     },
 
+    {
+        "name": "Macrumors",
+        "url": "https://macrumors.zendesk.com/hc/en-us/articles/201260797-How-do-I-cancel-my-account-",
+        "difficulty": "impossible",
```
Comment (tupaschoal) [inline _data/sites.json:6955]:
```suggestion
        "difficulty": "impossible",
```

**Comment by tupaschoal** [comment]
Thanks for your contribution!

**Pull request merged**